appcues/src/main/java/com/appcues/debugger/DebuggerViewModel.kt

```diff
@@ -210,16 +213,15 @@ internal class DebuggerViewModel(override val scope: AppcuesScope, debugMode: De
             // saving a capture is only valid in screen capture mode with token
             is ScreenCapture -> {
                 viewModelScope.launch {
-                    when (val result = screenCaptureProcessor.save(capture, currentMode.token)) {
+                    when (saveCaptureUseCase(currentMode.token, capture)) {
                         is Success -> _toastState.value = Rendering(
-                            type = ScreenCaptureSuccess(result.value) {
+                            type = ScreenCaptureSuccess(currentMode.token) {
```

Comment:
is the argument to `ScreenCaptureSuccess(..)` supposed to be the screen display name to show in the toast? This usage of `currentMode.token` confused me.

this may be the cause of the test failure https://app.circleci.com/pipelines/github/appcues/appcues-mobile-experience-spec/984/workflows/74f6348a-d294-4b5b-bfa0-747bc55f6dba/jobs/1202

Reply:
I did a merge on top of recent changes and this was an overlook, thanks!